### DISCUSSION TITLE
Switch to other project

### DISCUSSION THREAD

**Discussion by andim27** [supabase/supabase#33016] [Questions]
Flutter supabase error if switch to other project:

 - Supabase CLI 1.200.3
 - supabase_flutter: ^2.6.0
 
`supabase projects list:`
-show list of projects I can see which is LINKED

`supabase link --project-ref  "xxx":`
- error failed to load .env:
- 
Question: which .env is trying to load?
Is it necessary to link other project for flutter-supabase lib?

If works with one project all is ok -app start
if use credential from other project(under one account) then app stops:

```
D/TrafficStats( 4340): tagSocket(134) with statsTag=0xffffffff, statsUid=-1
D/ProfileInstaller( 4340): Installing profile for "my app"
```
No any answer 
What to do?

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.
